components/cross/Cross.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import Cross from "./Cross";
+
+export default {
+  title: "Cross",
```

Comment:
Cómo esto va a ser un ícono, lo podemos agrupar con los demás íconos que va a haber. Para eso podes poner en el título algo como "Iconos/NombreDelIcono" https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping

Reply:
Bien, ahí lo modifiqué. Lo puse dentro de una carpeta Icons.